README.md

```diff
@@ -1,2 +1,13 @@
 # Ansible-WSL
 Provisioning your Windows by Ansible from WSL
+
+## Getting Started
+
+### Prerequisites
+First, follow the Ansible offical setup of Windows hosts:
+https://docs.ansible.com/ansible/latest/user_guide/windows.html
+
+Next, get the WSL to run Ansible provisioning:
+https://www.microsoft.com/ja-jp/p/ubuntu-1804-lts/9n9tngvndl3q
+(This project only supports Ubuntu).
+After Ubuntu setup in WSL, install Ansible by apt: `apt install ansible`.
```

Comment:
If the latest version is preferred, use PPA as this guide:
https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#latest-releases-via-apt-ubuntu

Reply:
Make sure the installed Ansible's version is `> 2.0`
> `ansible --version`